infra/tool/create_configs.dart

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env dart
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//  http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:path/path.dart' as p;
+
+const exampleApps = {
+  Category.analytics: 'packages/analytics/amplify_analytics_pinpoint/example',
+  Category.api: 'packages/api/amplify_api/example',
+  Category.auth: 'packages/auth/amplify_auth_cognito/example',
+  Category.dataStore: 'packages/amplify_datastore/example',
+  Category.storage: 'packages/storage/amplify_storage_s3/example',
+};
+
+late final Directory repoRoot = () {
+  Directory dir = Directory.current;
+  while (p.absolute(dir.parent.path) != p.absolute(dir.path)) {
+    final files = dir.listSync().whereType<File>();
+    if (files.any((f) => p.basename(f.path) == 'aft.yaml')) {
+      return dir;
+    }
+    dir = dir.parent;
+  }
+  throw StateError('Could not locate repo root');
+}();
+
+void main() {
+  final outputs = jsonDecode(File('outputs.json').readAsStringSync())
+      as Map<String, dynamic>;
+  final categories =
```

Comment:
I got an error here:

```
Unhandled exception:
type 'Null' is not a subtype of type 'Map<String, dynamic>' in type cast
#0      main (file:///Users/tshepp/amplify_src/amplify-flutter/infra/tool/create_configs.dart:45:43)
#1      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297:19)
#2      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```

Reply:
It was empty object when I did this. I had deleted the stack in cloudformation from prev deploy and deleted everything locally before starting this.

Reply:
got this working w lates updates